lib/ex_machina/ecto.ex

```diff
@@ -122,6 +122,7 @@ defmodule ExMachina.Ecto do
     |> Map.delete(:__meta__)
     |> Map.drop(struct.__schema__(:associations))
     |> drop_autogenerated_ids(struct)
+    |> drop_nil_values
```

Comment:
Instead of adding this to `drop_ecto_fields` what do you think of adding it as the last pipe in `params_for` and `params_with_assocs`? I think that's more clear than adding it to `drop_ecto_fields`